bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/SolaxLocalAccessHandler.java

```diff
@@ -162,10 +288,28 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     @Override
     public void dispose() {
         super.dispose();
+        cancelSchedule();
+    }
+
+    private void cancelSchedule() {
         ScheduledFuture<?> schedule = this.schedule;
         if (schedule != null) {
             schedule.cancel(true);
             this.schedule = null;
         }
     }
+
+    private <T extends Quantity<T>> void updateChannel(String channelID, double value, Unit<T> unit,
+            Set<String> supportedChannels) {
+        if (supportedChannels.contains(channelID)) {
+            if (value > Short.MIN_VALUE) {
```

Comment:
Some of the values returned can be `Integer.MIN_VALUE`, for example `inverterData.getPV1Power()`. Since `Short.MIN_VALUE` > `Integer.MIN_VALUE`, values between those two constants would be considered out of range. Maybe you don't expect values in this range? In that case it will work, but could be more explicit.

Reply:
Yes. It is not expected to return lower than Short.MIN_VALUE because the raw data consists of Shorts byte array. Anyway it's not clear enough as this happens in another section of the code, so I changed all the default values to the Short.MIN_VALUE.